.gitignore

```diff
@@ -1 +1,2 @@
-*txt
\ No newline at end of file
+*txt
+test
```

Comment:
- [ ] [REQUIRED] One of the requirements states that test.md file and `node_modules` directory should be ignored by `.gitignore`. Please go-ahead to add them.

Reply:
mode_modules and test.md files added to .gitignore file